lib/ffmpegCommandService.js

```diff
@@ -28,9 +28,10 @@ class FfmpegCommandService {
      */
     createFfmpegCommand(command, allowUnknown = false) {
         const schema = this._createCommandSchema();
-        this._validateCommandSchema(command, schema, allowUnknown);
-
         const commandInArrayFormat = this._mapCommandToArrayFormat(command);
+
+        this._validateCommandSchema(commandInArrayFormat, schema, allowUnknown);
```

Comment:
Nice clean up 👍

Reply:
Problem with this inversion is that the error message won't be about the original object passed.

Reply:
@jonathan-fulton what do you mean?

Reply:
I think you're saying, we're going to get an error on the new array, even if someone passed in an object. I don't think that's a gigantic issue since it's still validating correctly; Ultimately, we can just decide to only support array form.